polaris-react/src/components/Scrollable/Scrollable.scss

```diff
@@ -10,6 +10,8 @@
   -webkit-overflow-scrolling: touch;
   position: relative;
   max-height: var(--pc-scrollable-max-height);
+  overflow-x: hidden;
+  overflow-y: hidden;
```

Comment:
These are overridden automatically by giving default values to horizontal and vertical props and assigning these [classes below](https://github.com/Shopify/polaris/compare/fix-scrollable-horizontal-bug?expand=1#diff-4f72e09af1aeed3cd3aac6646f2ac369d4eb8ab5624e64bd9ef8520a325673dbR22-R28)